drivers/storage/portworx/preflight.go

```diff
@@ -358,9 +360,18 @@ func (u *preFlightPortworx) processNodesChecks(recorder record.EventRecorder, st
 				k8sutil.InfoEvent(recorder, u.cluster, util.PassPreFlight, msg)
 				continue
 			}
+
+			passed = false // pre-flight status check failed, keep going for logging
+
+			if check.Result == "fatal" {
+				// collect fatal msgs for output at end.
+				fmsgs = append(fmsgs, check.Reason)
+				isFatal = true
+				continue
```

Comment:
Why do we `continue` instead of letting line 373 and 374 to log the failure out?

Reply:
Want to make sure that all the fatal events are logged at once.. will add a better comment.